### ISSUE TITLE
Error message during build should be more explicit

### ISSUE THREAD

**Issue by NicolasRannou** [Polymer/polymer-cli#297]
### Description
Error message during build should be more explicit

#### Expected Results

It should tell us which file is problematic.

#### Actual Results
```
error:   Uncaught exception: TypeError: Cannot read property 'is' of null
error:   TypeError: Cannot read property 'is' of null
    at Parser.parseDeclarationOrRuleset (/usr/local/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:191:33)
    at Parser.parseRule (/usr/local/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:80:21)
    at Parser.parseRules (/usr/local/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:62:25)
    at Parser.parseStylesheet (/usr/local/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:55:47)
    at Parser.parse (/usr/local/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:51:19)
    at css (/usr/local/lib/node_modules/polymer-cli/node_modules/css-slam/index.js:51:39)
    at dom5.queryAll.forEach.styleNode (/usr/local/lib/node_modules/polymer-cli/node_modules/css-slam/index.js:45:36)
    at Array.forEach (native)
    at html (/usr/local/lib/node_modules/polymer-cli/node_modules/css-slam/index.js:43:37)
    at GulpTransform._transform (/usr/local/lib/node_modules/polymer-cli/node_modules/css-slam/index.js:66:36)
```

Is it possible to add this kind of information with the current infrastructure?

**Comment by mikael2933** [comment]
Check files through, I think you forgot a `,` or you added one too many.

**Comment by NicolasRannou** [comment]
Yes - I found the issue it was a typo in the css as the error message suggested but I would hope polymer-cli could tell us directly which file.

In general it is fine to track it down if you commit small portions of code and run CI on every commit though.

**Comment by FredKSchott** [comment]
Fixed in [v0.13.0](https://github.com/Polymer/polymer-cli/releases/tag/v0.13.0)

**Comment by FredKSchott** [comment]
Fixed in [v0.13.0](https://github.com/Polymer/polymer-cli/releases/tag/v0.13.0)
